server/jobs/onboarding/email.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
+import { developerEmailTransport, opsEmailTransport } from '../../modules/email/emailModule';
+/**
+ * @desc Automates developer onboarding on email
+ *
+ * @param {array} placement Placement record whose developer is to be offboarded
+ * @param {object} automationResult Result of automation job
+ * @returns {void}
```

Comment:
Please update the JSDoc here. `placement` is an object and not an `array`.
The function also returns `undefined`(not returning anything) and not `void`.

Reply:
I have resolved it